demos/gallery/.vscode/launch.json

```diff
@@ -12,11 +12,22 @@
             ],
             "webRoot": "${workspaceRoot}"
         },
+        {
+            "name": "gallery/layout",
+            "type": "chrome",
+            "request": "launch",
+            "url": "file:///${workspaceRoot}/gallery.html?./samples/layout",
+            "runtimeArgs": [
+                "--allow-file-access-from-files",
+                "--disable-web-security"
+            ],
+            "webRoot": "${workspaceRoot}"
+        },
         {
             "name": "galleryItem",
             "type": "chrome",
             "request": "launch",
-            "url": "file:///${workspaceRoot}/galleryItem.html?./samples/geospatial/Leaflet/Countries.js",
+            "url": "file:///${workspaceRoot}/galleryItem.html?./samples/layout/Chart Panel.js",
```

Comment:
Extra space in filename?

Reply:
Its correct - there is a space in the filename